CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.8.5
```

Comment:
This should say [Next] (We really should introduce melos on this repo too...)

Reply:
fixed